packages/vuepress-theme-odyssey/styles/templates/_DocsTemplateComponent.scss

```diff
@@ -37,13 +37,9 @@
 
   a {
     display: flex;
-    padding: $spacing-xs-em $spacing-s-em $spacing-xs-em 0;
+    align-items: center;
+    padding: $spacing-xs-em;
     line-height: $base-line-height;
-
-    &:focus {
```

Comment:
Good call - link focus is one state I'm not super fond of overriding, even if it's an accessible solution.